inc/Config/QueryContext/HttpQueryContext.php

```diff
@@ -37,6 +37,10 @@ class HttpQueryContext implements QueryContextInterface, HttpQueryContextInterfa
 							'type' => 'array',
 							'required' => false,
 						],
+						'generate' => [
```

Comment:
Thought about naming this as `transform` or `filter`, but ended up naming it as `generate` to match with name for similar field in the `output_schema` - https://github.com/Automattic/remote-data-blocks/blob/fa91a56f6b36a3dc72390ef7bd704a862cf2b613/inc/Config/QueryContext/HttpQueryContext.php#L61-L64.

Wouldn't mind using another name if there is an good reason to do it.